docs/user/environment-variables.rst

```diff
@@ -1,116 +1,127 @@
-Environment Variables
-=====================
+.. _Environment Variables:
 
-Read the Docs supports two types of environment variables in project builds:
+Understanding environment variables
+===================================
 
-* `Default environment variables`_
-* `User-defined environment variables`_
+Read the Docs allows you to define your own environment variables to be used in the build process.
+This is useful for adding build secrets such as API tokens.
```

Comment:
```suggestion
Read the Docs allows you to define your own environment variables to be used in the build process.
It also defines a set of :doc:`default environment variables</reference/environment-variables>` with information about your build.
These are useful for different purposes:

* Custom environment variables are useful for adding build secrets such as API tokens.
* Default environment variables are useful for varying your build specifically for Read the Docs or specific types of builds on Read the Docs. 
```

Reply:
I don't feel too strongly. We use `default` in a _lot_ of places (eg. `default` values of things). I think `default` makes sense if they can be overwritten, but I don't think that's common here, so `pre-defined` is probably better. Or even `Build-specific environment variables` to be even more explicit and useful (who cares if they are pre-defined, what _are_ they?)